packages/wallets/torus/src/adapter.ts

```diff
@@ -150,21 +136,38 @@ export class TorusWalletAdapter extends BaseSignerWalletAdapter {
 
     async signAllTransactions(transactions: Transaction[]): Promise<Transaction[]> {
         try {
-            const keypair = this._keypair;
-            if (!keypair) throw new WalletNotConnectedError();
+            const wallet = this._torus;
+            if (!wallet) throw new WalletNotConnectedError();
 
             try {
-                for (const transaction of transactions) {
-                    transaction.partialSign(keypair);
-                }
+                return (await wallet.signAllTransactions(transactions)) || transactions;
             } catch (error: any) {
                 throw new WalletSignTransactionError(error?.message, error);
             }
+        } catch (error: any) {
+            this.emit('error', error);
+            throw error;
+        }
+    }
 
-            return transactions;
+    async signMessage(message: Uint8Array): Promise<Uint8Array> {
+        try {
+            const wallet = this._torus;
+            if (!wallet) throw new WalletNotConnectedError();
+
+            try {
+                const signature = await wallet.signMessage(message);
+                return Uint8Array.from(signature);
```

Comment:
What's the return type of `Torus::signMessage`? Is it necessary to cast with `Uint8Array.from`?